tests/designs/avalon_module/Makefile

```
@@ -39,7 +39,7 @@ else
 
 TOPLEVEL := burst_read_master
 
-COCOTB?=$(PWD)/../../..
+COCOTB?=$(shell pwd)/../../..
```

Would prefer we stay consistent with all the other files

```suggestion
PWD=$(shell pwd)

COCOTB?=$(PWD)/../../..
```

That way we can do further cleanup with a find and replace